jsquiz.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
It's standard convention to name this file `index.html`. That's what http-server is looking for by default when it runs.